chia/farmer/farmer.py

```diff
@@ -670,6 +702,32 @@ def get_receiver(self, node_id: bytes32) -> Receiver:
             raise KeyError(f"Receiver missing for {node_id}")
         return receiver
 
+    def increment_pool_stats(
+        self,
+        p2_singleton_puzzle_hash: bytes32,
+        name: str,
+        count: int = 1,
+        value: Any = None,
```

Comment:
it's a bit unfortunate that we use a dictionary instead of a class for `pool_state`, so we lose type information. but it looks like this `Any` could be `int`, `ErrorResponse` and `Dict[str, Any]`. I would think we want to make it `object`

Reply:
Done